platform.md

```diff
@@ -461,12 +461,22 @@ Usage:
       - All run-image config values SHALL be preserved unless this conflict with another requirement
     - MUST contain all buildpack-provided launch layers as determined by the [Buildpack Interface Specfication](buildpack.md)
     - MUST contain one or more app layers as determined by the [Buildpack Interface Specfication](buildpack.md)
-    - MUST contain a layer that includes `<launcher>`
+    - MUST contain a launcher layer containing:
+        - A file with the contents of the `<launcher>` file at path `/cnb/lifecycle/launcher`
+        - One symlink per buildpack provided `process` with name `/cnb/process/<type>` and target `/cnb/lifecycle/launcher`
     - MUST contain a layer that includes `<layers>/config/metadata.toml`
-    - MUST have `ENTRYPOINT=<launcher>`
+    - **If** `<process-type>` matches a buildpack-provided process:
+      - MUST have `ENTRYPOINT=/cnb/process/<process-type>`
+    - **If** `<process-type>` does not match a buildpack-provided process:
+      - **If** there is exactly one buildpack-provided process
```

Comment:
Is it possible that there won't be any buildpack-provided processes?
If there is such situation, maybe we can change it to an `if-else` statements.